app/src/neptune_eye/object_detection/yolo_object_detection.py

```
@@ -0,0 +1,174 @@
+        
+        try:
+            self._model = YOLO(absolute_model_path, task='detect')
+        except Exception as e:
+            raise RuntimeError(f"Failed to load YOLO model from {absolute_model_path}: {e}") from e
+        
+    def detect(self, frame: Any) -> List:
+        """
+       
+        """
+        if self._model is None:
+            raise RuntimeError("Model not set up. Call setup() before detect().")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)